src/view/terminalview.cpp

```diff
@@ -51,6 +57,21 @@ std::string TerminalView::stringifyFood(Food & food) {
     return outputString;
 };
 
-std::string TerminalView::stringifyEntity(Entity & entity) {
+std::string TerminalView::stringifyIndividual(Individual * individual) {
```

Comment:
instead of creating stringifyFood and stringifyIndividual you could also keep stringifyEntity and overload it with Food or Individual
(or, for even more generic naming, call it stringify only)

Reply:
I considered that since the functions are quite similar. But I might have forgotten even though I wrote a reminder comment above them

Reply:
I think why I didn't do it was because the individual also gives the ID. But still I guess I could just do it by overloading

Reply:
that's no reason to not overload it.
Overloading means you write different functions depending on the input, so in stringify(Individual) you can print everything plus ID, and in stringify(Food) you only print position and food

Reply:
Tbh, you could just have one function stringify(Object) and read out what specific type it is - saving you even more duplicate code XD

Reply:
I'm looking forward to see solve this without writing much duplicate code

Reply:
commit 46ca073238829b9b1c997422f648d658927a3ac5